src/leapYear.ts

```diff
@@ -0,0 +1,13 @@
+const leapYear = (inputYear: number): boolean => {
```

Comment:
I think I preferred it when this was called `isLeapYear`
As its a function, I try to keep all function names as verbs as this describes them better